app/graphql/mutations/create_task.rb

```diff
@@ -13,9 +13,11 @@ def resolve(title:, points:, description:, taskListId:)
         # TODO: Change board manager to current user
         task_list = TaskList.find(taskListId)
         board_manager = task_list.board.author
+        new_order_position = task_list.tasks.count + 1
         Task.create!(title: title, points: points, 
                     description: description, task_list_id: taskListId,
-                    creator_id: board_manager.id, completed: false)
+                    creator_id: board_manager.id, completed: false, 
+                    order: new_order_position)
```

Comment:
If you will use `create!` with a bang operator in order to yield an exception; I recommend using `.new().save! `